### ISSUE TITLE
CentOS 7 install from repository fully broken

### ISSUE THREAD

**Issue by minddrive** [arnehilmann/yumrepos#6]
The 'Installation' instructions to install the software on this page: https://arnehilmann.github.io/yumrepos/index.html does not work.

From a CentOS 7.4 Docker image, following these instructions ends up with the following:
```
# yum install yumrepos-behind-nginx-on-centos7
[...]
---> Package yumrepos.noarch 0:0.9.10-py35 will be installed
--> Processing Dependency: python(abi) = 3.5 for package: yumrepos-0.9.10-py35.noarch
--> Processing Dependency: python3-flask for package: yumrepos-0.9.10-py35.noarch
--> Finished Dependency Resolution
Error: Package: yumrepos-0.9.10-py35.noarch (yumrepos)
           Requires: python(abi) = 3.5
           Installed: python-2.7.5-58.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Available: python34-3.4.5-4.el7.x86_64 (epel)
               python(abi) = 3.4
Error: Package: yumrepos-0.9.10-py35.noarch (yumrepos)
           Requires: python(abi) = 3.5
           Installed: python-2.7.5-58.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Installing: python34-3.4.5-4.el7.x86_64 (epel)
               python(abi) = 3.4
Error: Package: yumrepos-0.9.10-py35.noarch (yumrepos)
           Requires: python3-flask
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

Why is it trying to install the Fedora 24 version of the RPM on CentOS 7?  It would be nice to know if there's a workaround to get it to install the correct version.

**Comment by arnehilmann** [comment]
You are right, the installation as described at the moment fails (and feels) miserably!
For some reason, yum "decides" to use the py35 version and not the (also available and correct) py27 version... *grr*

as a temporary workaround: install the correct yumrepos version first. 
For your case, in a fresh centos7 container and as root user (note the 3rd step):

`yum install epel-release yum-utils`
`yum-config-manager --add-repo https://arnehilmann.github.io/yumrepos/yumrepos.repo`
`yum install yumrepos-0.9.10-py27`
`yum install yumrepos-behind-nginx-on-centos7`

In the meantime, I will have a closer look into that resolving issue...

**Comment by minddrive** [comment]
Thank you for the quick response!  Given I am running this in a temporary Docker container (creating repositories to be uploaded to AWS S3), I made some minor tweaks to the developer install and run that is currently working... I think... hit maybe another strange bug that I'll open another ticket on. :)  I will give the workaround a try in the near future, though!
